CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/MyCoinView.swift

```diff
@@ -51,7 +51,7 @@ private extension MyCoinView {
             // TODO: - Remove dummy
             TimeFrameSection()
             CoinStatisticsSection(
-                coinDetailItem: CoinDetailItem(
+                coinDetailItem: viewModel.coinDetail ?? CoinDetailItem(
```

Comment:
Why are we falling back to the mock data if the real data is nil? This could create confusion to the user that we saw large price fluctuation here compared to real data from other sources, especially for coins with small values like dogecoin, ripple, etc. I think we should have another default data that shows all zero values or maybe hide the section completely if it is nil. What do you think? 🤔

Reply:
Thank you for the raise 🙏 .
>I think we should have another default data that shows all zero values or maybe hide the section completely if it is nil

These are good suggestions 👍 . But I think when the real data is nil, it means that "There is no data for this". Therefore, I come up with setting it's nil data with "-". I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/96/commits/e998377324a2ba308e06efc5cfcd5a69c11ac58f. What do you think about this approach? Thanks.

Reply:
Showing the nil data as `"-"` is a good move too 👍